cpanfile

```diff
@@ -1,4 +1,4 @@
-requires 'perl', '5.012000';
+requires 'perl', '5.022000';
```

Comment:
I don't hope to drop older perl support, because still we have 5.16 project 😭 
I'll only stop testing on CI with older perl versions and keep cpanfile requires perl version as 5.12.